qir/qat/StaticResourceComponent/QubitRemapPass.cpp

```diff
@@ -84,7 +84,7 @@ namespace quantum
                 }
                 break;
             case AllocationAnalysis::ResultResource:
-                if (results_mapping.find(index) == results_mapping.end())
+                if (results_mapping.find(index) != results_mapping.end())
```

Comment:
Why isn't there a unit test that caught this? It would be good to add a test - that seems like a gap.

Reply:
Answered offline: The Result handling here is set up following the same pattern as for Qubits, except we don't have a Reset for Results, and hence no tests.

Reply:
Answered offline: The Result handling here is set up following the same pattern as for Qubits, except we don't have a Reset for Results, and hence no tests.